openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -326,3 +337,31 @@ def test_access_based_on_track(self, user_track, component_type, course, is_gate
             course_id=self.courses[course]['course'].id,
             is_gated=is_gated
         )
+
+    @ddt.data(
+        ('problem', 'problem','audit', 404),
+        ('problem', 'problem','verified', 200),
+        ('problem', 'problem_2', 'audit', 200),
+        ('problem', 'problem_2', 'verified', 200),
+    )
+    @ddt.unpack
+    def test_xblock_handlers(self, xblock_type, xblock_name, user, status_code):
+        """
+        Test the ajax calls to the problem xblock to ensure the LMS is sending back
+        the expected response codes on requests when content is gated for audit users
+        (404) and when it is available to audit users (200). Content is always available
+        to verified users.
+        """
+        problem_location = self.course.id.make_usage_key(xblock_type, xblock_name)
```

Comment:
I would use this for the usage_id block.scope_ids.usage_id (assuming it also works) since that is what we are using in other tests in this file

Reply:
Since I'm not grabbing the block directly I think I'd still like to make use of this function here.